ir/src/main/scala/org/scalajs/core/ir/IRVersionNotSupportedException.scala

```diff
@@ -0,0 +1,14 @@
+package org.scalajs.core.ir
+
+import java.io.IOException
+
+object IRVersionNotSupportedException {
+  def mkMessage(version: String, supported: Set[String]): String = {
+    s"This version ($version) of Scala.js IR is not supported. " +
+    s"Supported versions are: ${supported.mkString(", ")}"
+  }
+}
+
+case class IRVersionNotSupportedException(version: String, supported: Set[String])
+    extends IOException(
+      IRVersionNotSupportedException.mkMessage(version, supported))
```

Comment:
Also the indentation is not right. It should be 4 spaces from the beginning of the line. Better would be not to break the `extends` line.